macros/answers/PGstringevaluators.pl

```diff
@@ -312,66 +305,62 @@ sub str_cmp {
 	return (wantarray) ? @output_list : $output_list[0];
 }
 
-=head1 "mode"_str_cmp functions
+=head2 std_str_cmp
 
 The functions of the form "mode"_str_cmp() use different functions to
 specify which filters to apply. They take no options except the correct string.
 There are also versions which accept a list of strings.
 
-=over
-
-=item standard
-
-	std_str_cmp($correctString)
-	std_str_cmp_list(@correctStringList)
+    std_str_cmp($correctString)
+    std_str_cmp_list(@correctStringList)
 
 Filters: compress_whitespace, ignore_case
 
-=item standard, case sensitive
+=head2 std_cs_str_cmp
 
-	std_cs_str_cmp($correctString)
-	std_cs_str_cmp_list(@correctStringList)
+standard, case sensitive
+
+    std_cs_str_cmp($correctString)
+    std_cs_str_cmp_list(@correctStringList)
 
 Filters: compress_whitespace
 
-=item strict
+=head2 strict_str_cmp
 
-	strict_str_cmp($correctString)
-	strict_str_cmp_list(@correctStringList)
+    strict_str_cmp($correctString)
+    strict_str_cmp_list(@correctStringList)
 
 Filters: trim_whitespace
 
-=item unordered
+=head2 unordered_str_cmp
 
-	unordered_str_cmp( $correctString )
-	unordered_str_cmp_list( @correctStringList )
+    unordered_str_cmp( $correctString )
+    unordered_str_cmp_list( @correctStringList )
 
 Filters: ignore_order, ignore_case
 
-=item unordered, case sensitive
+=head2 unordered_cs_str_cmp
 
-	unordered_cs_str_cmp( $correctString )
-	unordered_cs_str_cmp_list( @correctStringList )
+    unordered_cs_str_cmp( $correctString )
+    unordered_cs_str_cmp_list( @correctStringList )
 
 Filters: ignore_order
 
-=item ordered
+=head2 ordered_str_cmp, ordered_str_cmp_list
 
-	ordered_str_cmp( $correctString )
-	ordered_str_cmp_list( @correctStringList )
+    ordered_str_cmp( $correctString )
+    ordered_str_cmp_list( @correctStringList )
 
 Filters: remove_whitespace, ignore_case
 
-=item ordered, case sensitive
+=head2 ordered_cs_str_cmp, ordered_cs_str_cmp_list
 
-	ordered_cs_str_cmp( $correctString )
-	ordered_cs_str_cmp_list( @correctStringList )
+    ordered_cs_str_cmp( $correctString )
+    ordered_cs_str_cmp_list( @correctStringList )
 
 Filters: remove_whitespace
 
-=back
-
-=head2 Examples
+=head3 Examples
```

Comment:
This shouldn't be `head3`.  That mean that this header is under the previous `head2`.  So this is saying that the following are examples of using the `ordered_cs_str_cmp` and `ordered_cs_str_cmp_list` methods, and this is not what these are examples of.  These are examples of many of the previous methods before that as well.